model/collection.py

```diff
@@ -97,6 +98,11 @@ class Collection(Base, HasFullTableCache):
         Integer, ForeignKey('externalintegrations.id'), nullable=True
     )
 
+    # When deleting a collection, this flag is set to True so that the deletion
+    # script can take care of deleting it in the background. This is
+    # useful for deleting large collections which can timeout when deleting.
+    set_to_delete = Column(Boolean, default=False)
+
```

Comment:
This has what I think is your old name for the field -- change it to `marked_for_deletion` and we should be good.

Reply:
Missed that.